pkg/cmd/cli/backuplocation/set.go

```diff
@@ -106,6 +112,17 @@ func (o *SetOptions) Run(c *cobra.Command, f client.Factory) error {
 	}
 
 	location.Spec.Default = o.DefaultBackupStorageLocation
+
```

Comment:
Should we add the check to limit only one key-value pair?
```go
        if len(o.Credential.Data()) > 1 {
		return errors.New("--credential can only contain 1 key/value pair")
	}
```

Reply:
I'd like this error too if we're just going to process one set.